plugins/ruby-foreman-tasks/bin/foreman-tasks

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env ruby
```

Comment:
This might be a problem on 12.04, should we use the foreman-ruby symlink we installed as the shebang line?